lightkurve/tests/test_synthetic_data.py

```diff
@@ -0,0 +1,188 @@
+from __future__ import division, print_function
+
+import numpy as np
+
+from astropy.utils.data import get_pkg_data_filename
+import pytest
+import warnings
+
+from ..targetpixelfile import KeplerTargetPixelFile, TessTargetPixelFile
+from ..correctors import SFFCorrector, PLDCorrector
+
+filename_synthetic_sine = get_pkg_data_filename("data/synthetic/synthetic-k2-sinusoid.targ.fits.gz")
+filename_synthetic_transit = get_pkg_data_filename("data/synthetic/synthetic-k2-planet.targ.fits.gz")
+filename_synthetic_flat = get_pkg_data_filename("data/synthetic/synthetic-k2-flat.targ.fits.gz")
+
+
+lacks_BLS = False
+try:
+    from astropy.stats.bls import BoxLeastSquares
+except ImportError:
+    lacks_BLS = True
+
+def test_sine_SFF():
```

Comment:
PEP8 also stipulates `lower_case_with_underscores` for all function names.  It's good to be consistent because then we don't have to think about case.